### PR TITLE
Fix styling of sui on epochs page

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
## Description

Fixes the styling caused by using old components, and mark those as deprecated.

## Test Plan

Ran locally.

### CODE REVIEW

**Comment by Jordan-Mysten** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10701** <a href="https://app.graphite.dev/github/pr/MystenLabs/sui/10701" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/MystenLabs/sui/10701?utm_source=stack-comment).

File: apps/explorer/src/components/transactions/TxCardUtils.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ import styles from './RecentTxCard.module.css';
 
 import { AddressLink, TransactionLink } from '~/ui/InternalLink';
 
+/** @deprecated This styling is not correct, use `<Text>` to create the correctly styled components */
```
Comment (williamrobertson13) [inline apps/explorer/src/components/transactions/TxCardUtils.tsx:23]: might be nice to add a ticket to the backlog for removing the remaining usages of `SuiAmount` so it's tracked somewhere 😄

**Review by williamrobertson13 — APPROVED** [review]
lgtm

File: apps/explorer/src/pages/epochs/EpochDetail.tsx [hunk 2]
```diff
@@ -66,54 +82,48 @@ export default function EpochDetail() {
...
+                    <SuiStats
+                        label="Total Rewards"
+                        tooltip="Total Rewards"
+                        amount={
+                            epochData?.endOfEpochInfo
+                                ?.totalStakeRewardsDistributed
+                        }
+                    />
 
                     <Stats
```
Comment (mamos-mysten) [inline apps/explorer/src/pages/epochs/EpochDetail.tsx:120]: this should also be `SuiStats`

rip `SuiAmount` 🕊️

**Pull request merged**